packages/admin/cms-admin/src/generator/future/generateForm/generateFormLayout.ts

```diff
@@ -0,0 +1,53 @@
+import { FormConfig, FormLayoutConfig } from "../generator";
+import { Imports } from "../utils/generateImportsCode";
+
+export function generateFormLayout(
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    config: FormLayoutConfig<any>,
+    fieldsCode: string,
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    formConfig: FormConfig<any>,
+) {
+    const gqlType = formConfig.gqlType;
+    const instanceGqlType = gqlType[0].toLowerCase() + gqlType.substring(1);
+
+    const imports: Imports = [];
+    let code = "";
+    const formFragmentFields: string[] = [];
```

Comment:
This isn't used at the moment... Is this for follow up tasks?

Reply:
ah, yes. I missed that, in the refactoring the return structure of generateFormLayout and generateFormField are the same.